submit.go

```diff
@@ -77,15 +77,15 @@ type jsonLogEntry struct {
 	Lines string `json:"lines"`
 }
 
-
 type genericWebhookPayload struct {
 	parsedPayload
-	ReportURL string             `json:"report_url"`
-	ListingURL string            `json:"listing_url"`
+	ReportURL  string `json:"report_url"`
+	ListingURL string `json:"listing_url"`
 }
 
 // the payload after parsing
 type parsedPayload struct {
+	ID         string            `json:"id"`
```

Comment:
a comment here describing the function of the field might be nice.

Reply:
Have tried to comment on all fields now.